components/ScrollToTop.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState, useEffect } from 'react'
+import '../public/styles/main.scss'
+import { Flex, PseudoBox, IconButton } from '@chakra-ui/core'
+
+const ScrollToTop = () => {
+  const [show, setShow] = useState(false)
+
+  useEffect(() => {
+    const scrollFunc = () => {
+      if (window.scrollY > 400) {
+        setShow(true)
+      } else {
+        setShow(false)
+      }
+    }
+    window.addEventListener('scroll', scrollFunc)
+  }, [])
+
+  const scrollBackFn = () => {
+    window.scrollTo({ top: 0, behavior: 'smooth' })
+  }
+
+  return (
+    <Flex>
+      <PseudoBox _hover={{ color: '#f6b333' }}>
+        <IconButton
+          icon='chevron-up'
+          variantColor='black'
+          variant='outline'
+          aria-label='Top'
+          fontSize='25px'
+          onClick={scrollBackFn}
+          position='fixed'
+          borderRadius='50%'
+          bottom='200px'
```

Comment:
On mobile, this is a little high, could we try bottom={['50px', '200px']} ? that might look better on mobile 

what this does is designate 50px for all screen sizes, and then above medium breakpoint it becomes 200px

Reply:
@joelwass cool! that's nice...

Reply:
Could you make the change and test it out? TBH changing to bottom='50px' might work as well because 200px is quite high even on desktop

Reply:
@joelwass actually 50 is the right height but when the visitor reaches the bottom of the page the icon rests on the footer and both camouflage. Else I should exchange hover color with Icon variant color.

Reply:
Hmmmm that's a great point. up to you! whatever you think is best push it up and we can look at the change :)

Reply:
also, the camouflage can be ignored I suppose because the moment the user scrolls up the Icon shows up with the light background.

Reply:
I pushed the changes added retained 50 instead of 200

Reply:
Love it, thanks!